features/admin/ccs_admin_journey.feature

```diff
@@ -0,0 +1,45 @@
+@not-production @functional-test
+Feature: CCS admin user journey through Digital Marketplace
+
+Scenario: Setup for tests
+    Given I have test suppliers
+    And The test suppliers have declarations
+
+Scenario: As a CCS Sourcing user, I wish to search for supplier(s) by supplier name prefix
+  Given I have logged in to Digital Marketplace as a 'CCS Sourcing' user
+  When I enter 'DM Functional Test Supplier' in the 'supplier_name_prefix' field
+  And I click the search button for 'supplier_name_prefix'
+  Then I am presented with the 'Suppliers' page for all suppliers starting with 'DM Functional Test Supplier'
+
+Scenario: As a CCS Sourcing user I should be able to edit a supplier declaration
+    Given I am logged in as a 'CCS Sourcing' and navigated to the 'Suppliers' page by searching on suppliers by name prefix 'DM Functional Test Supplier'
+    When I click the 'G-Cloud 7 declaration' link for the supplier 'DM Functional Test Supplier'
+    Then I am presented with the admin G-Cloud 7 declaration page
+    When I click the 'Edit' link for 'G-Cloud 7 essentials'
+    And I choose 'No' for 'PR1'
```

Comment:
It might be worth checking the existing value of the fields before making the change - that way the test can be sure that the "Save" button really has saved a new change to the data.  e.g. a new step something like `And 'PR1' has the value 'Yes'` before the `And I choose...` step, and similarly for the `input-SQ3-1k` change.